scio-smb/src/main/scala/com/spotify/scio/smb/syntax/SortMergeBucketScioContextSyntax.scala

```diff
@@ -338,6 +332,9 @@ final class SortedBucketScioContext(@transient private val self: ScioContext) ex
     toR: CoGbkResult => R
   ) extends Serializable {
 
+    def withoutReiterable: SortMergeTransformReadBuilder[K, R] =
```

Comment:
This is the least intrusive for `sortMergeTransform` APIs and doesn't add a default arg that causes method overloading issues.

Doesn't work for `sortMergeCoGroup` though since they don't have intermediate type.
OTOH they don't offer much since co-group is a very thin layer. Maybe whoever needs this can just write Java?

Reply:
@ClaireMcGinty thoughts?

OTOH the single global flag via `PipelineOptions` won't work since `reiterable` must be true if it's used in a JOIN.